src/vs/workbench/parts/html/browser/webview.ts

```diff
@@ -145,29 +153,55 @@ export default class Webview {
 		::-webkit-scrollbar {
 			width: 14px;
 			height: 10px;
-		}
-		::-webkit-scrollbar-thumb:hover {
-			background-color: rgba(100, 100, 100, 0.7);
 		}`;
 
+		let bodyClasses = {
+			remove: ['monaco-editor', 'vs', 'vs-dark', 'hc-black']
```

Comment:
do we need `monaco-editor`? 
and for the sake of being aligned and explicit should we call the theme classes: `vscode-light`, `vscode-dark`, and `vscode-high-contrast`?